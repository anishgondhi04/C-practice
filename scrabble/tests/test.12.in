10
2
1 2 V hello
2 3 H hello
0

