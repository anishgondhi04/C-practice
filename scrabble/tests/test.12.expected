Invalid word placement: (2,3) H, hello
..........
..........
.h........
.e........
.l........
.l........
.o........
..........
..........
..........
